### DISCUSSION TITLE
Output of chime_tts: say

### DISCUSSION THREAD

**Discussion by dbullendsl** [nimroddolev/chime_tts#60] [Ideas]
I've noticed during my testing that if I use the chime_tts.say function selecting Volume Level, Join and Unjoin and choosing multiple Alexas that they will actually respond to the volume increase and the join function, but fail with the outputting of mp3. I'm wondering if there were a selector box to choose alexa vs non-alexa and change whatever is used to push the sound out to the appropriate box. In otherwords - best of both worlds

**Comment by nimroddolev** [comment]
> but fail with the outputting of mp3

Can you please clarify what you mean? Do you mean no audio is played by the speakers?

**Reply by dbullendsl** [reply]
I must have imagined this, I can't duplicate the results of simultaneous playback, I was using a different pair of Alexas. Amazon must have a trick to doing it because when playing Spotify over multiple devices they're all synchronized, but there's nothing in the Amazon Media Player integration. They say it's not possible.  Oh well.

Any way, the response from the Alexa devices is that they need to be supplied with a publicly  accessible URL. But the integration does change the volume.

**Reply by nimroddolev** [reply]
I don't think the sync issues are something I can fix with Chime TTS. Behind the scenes it uses the native Home Assistant service `media_player.play_media` for audio playback.

> Any way, the response from the Alexa devices is that they need to be supplied with a publicly accessible URL

Are you saying the public audio URL from `chime_tts.say_url` won't play on Alexa devices?

**Reply by dbullendsl** [reply]
no - was just trying to see what happened if I used the chime_tts.say function. Seemed like some of the functionality works but, not the output which of course for Alexa devices are different from everything else.  I'm not well versed in python but I tried looking at your code, but don't understand how the calls to the tts platforms are made. So I don't want to go down that rabbit hole, just observing.

**Reply by nimroddolev** [reply]
Hi Dave. It's been a while since you reported this issue, with v1.0.0 are you able to use `chime_tts.say` on your Alexas successfully?

**Reply by dbullendsl** [reply]
Sorry, been busy for the last few weeks, and actually I hadn't noticed any issues. I'm using the chime_tts.say function exclusively now so the answer is a definite yes. I should say not completely flawless because sometimes the volume doesn't revert back on my Alexa Echo Show but does work on all the Alexa Dots.  The Echo Show is a early model so maybe that's the problem - not worried about it.

**Comment by nimroddolev** [comment]
Excellent 👍
I’m closing the issue but please reopen it if you have the issue again.
